prebid-mobile/pbm-api/android/code-integration-android.md

```diff
@@ -306,6 +306,16 @@ To stop sending stored bid response signals use the following method:
 void clearStoredBidResponses()
 ```
 
+### AuctionSettingsId
+{:.no_toc}
+
+String to separate out account from "auction settings". This is used to set `ext.prebid.storedrequest.id`, otherwise prebidServerAccountId is taken by default.
```

Comment:
```suggestion
Allows you to separate account from "auction settings". This is used to set `ext.prebid.storedrequest.id`, otherwise prebidServerAccountId is taken by default. This allows each app to have different global parameters like timeout, price granularity, etc. Please work with your Prebid Server provider to determine what to enter here. 
```